h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -454,7 +454,7 @@ public void run() {
    * Each model can override this logic, based on parameters, dataset size, etc.
    * @return How many models to train in parallel during cross-validation
    */
-  protected int nModelsInParallel(int folds) {
+  public int nModelsInParallel(int folds) {
```

Comment:
this is only ment for cross validation, not when building the main model

Reply:
Yesyes.. discussed this with @michalkurka, that it should be ok to use it in this case. I just think I should put folds = 1. Or do you think I should do some different method for rulefit case based on this one?

Reply:
now there is `nTreeEnsemblesInParallel(int numDepths)`